cmd/app.go

```diff
@@ -46,6 +47,15 @@ func Main() {
 		klog.Infof("FLAG: --%s=%q", flag.Name, flag.Value)
 	})
 
+	// Process on macOS must run using sudo
+	if runtime.GOOS == "darwin" {
+		currentUser, _ := user.Current()
```

Comment:
we should handle the error, if we are not able to get the user we have two options:
- fail and exist
- warn and continue

The later seems safer but it may have a non-working deployment :(

Reply:
let's check the error and if it fails just log the warning and continue

Reply:
This could be simplified:

```go
if runtime.GOOD == "darwin" && sys.Geteuid != 0 {
   fmt.Println("Please run this again with `sudo`.")
}
```

Reply:
awesome, @zzzk1 can you implement this suggestion?

Reply:
> awesome, @zzzk1 can you implement this suggestion?

Of couse, i will